addons/sourcemod/scripting/ButtonNotifier.sp

```diff
@@ -3,32 +3,54 @@
 #include <sourcemod>
 #include <sdktools>
 #include <cstrike>
+#include <clientprefs>
 #include <multicolors>
-#tryinclude <entWatch>
+#undef REQUIRE_PLUGIN
+#tryinclude <EntWatch>
+#define REQUIRE_PLUGIN
 
 #pragma newdecls required
 
+#define CHAT 1
+#define CONSOLE 2
+
 ConVar g_cBlockSpam;
 ConVar g_cBlockSpamDelay;
-ConVar g_cPrintType;
 
+Handle g_hButtons = INVALID_HANDLE;
+Handle g_hTriggers = INVALID_HANDLE;
+
+int g_iButtonsDisplay[MAXPLAYERS+1];
+int g_iTriggersDisplay[MAXPLAYERS+1];
 int g_ilastButtonUse[MAXPLAYERS+1] = { -1, ... };
 int g_iwaitBeforeButtonUse[MAXPLAYERS+1] = { -1, ... };
 
 public Plugin myinfo =
 {
 	name = "Button Notifier",
-	author = "Silence, maxime1907",
+	author = "Silence, maxime1907, .Rushaway",
 	description = "Logs button and trigger presses to the chat.",
-	version = "1.0.0",
+	version = "2.0",
 	url = ""
 };
 
 public void OnPluginStart()
 {
+	/* CONVARS */
 	g_cBlockSpam = CreateConVar("sm_buttonnotifier_block_spam", "1", "Blocks spammers abusing certain buttons", FCVAR_NONE, true, 0.0, true, 1.0);
 	g_cBlockSpamDelay = CreateConVar("sm_buttonnotifier_block_spam_delay", "5", "Time to wait before notifying the next button press", FCVAR_NONE, true, 1.0, true, 60.0);
-	g_cPrintType = CreateConVar("sm_buttonnotifier_print_type", "0", "Print type of button. (0 = chat, 1 = console, 2 = server)", FCVAR_NONE, true, 0.0, true, 2.0);

```

Comment:
add two cvars to let server administrators set the default values that they want for new players:
- sm_buttonnotifier_print_type_button_default
- sm_buttonnotifier_print_type_trigger_default